gflownet/utils/logger.py

```diff
@@ -60,7 +61,7 @@ def __init__(
             if slurm_job_id:
                 wandb_config["slurm_job_id"] = slurm_job_id
             self.run = self.wandb.init(
-                config=wandb_config, project=project_name, name=run_name
+                config=wandb_config, project=project_name, name=run_name, notes=notes
```

Comment:
So this goes on the description section of the wandb overview, where it says something like "What is special about this run?"?

Reply:
Yes exactly

<img width="318" alt="SCR-20230925-oovt" src="https://github.com/alexhernandezgarcia/gflownet/assets/9283470/d1c45e03-3628-4ce2-8835-d2ed8b7b4a93">